### PR TITLE
PLU-89: Migrate flow substep

### PR DESCRIPTION
**Pull request by m0nggh**
## Problem

Flow substep still uses MUI, so it does not reflect correctly with Figma.

## Solution

Migrate to Chakra :-)

## Things changed
- Text style change to `subhead1`
- Align icons and change the icons
- Change the success/error icon to react icons
- Change padding top and bottom
- Change the hover and active for the collapse

## Before & After Screenshots

**BEFORE**:
image

**AFTER**:

https://github.com/opengovsg/plumber/assets/65110268/ab6d6bc3-ef73-4854-a2cc-f9a59b77d9ff

## Tests
Regression test:
- Check that I can still continue to next step
- Check that I can still test step and publish pipe and activate workflow

### CODE REVIEW

File: packages/frontend/src/components/FlowSubstepTitle/index.tsx [hunk 1]
```diff
@@ -13,8 +14,22 @@ type FlowSubstepTitleProps = {
...
 
-const validIcon = <CheckCircleIcon color="success" />
-const errorIcon = <ErrorIcon color="error" />
+const validIcon = (
+  <Icon
+    boxSize={6}
+    as={BiSolidCheckCircle}
+    color="interaction.success.default"
+    mr={4}
+  ></Icon>
```
Comment (pregnantboy) [inline packages/frontend/src/components/FlowSubstepTitle/index.tsx]: nit: 
```suggestion
  />
```

File: packages/frontend/src/components/FlowSubstepTitle/index.tsx [hunk 2]
```diff
@@ -13,8 +14,22 @@ type FlowSubstepTitleProps = {
...
+)
+const errorIcon = (
+  <Icon
+    boxSize={6}
+    as={BiSolidErrorCircle}
+    color="interaction.warning.default"
+    mr={4}
+  ></Icon>
```
Comment (pregnantboy) [inline packages/frontend/src/components/FlowSubstepTitle/index.tsx]: same for this

File: packages/frontend/src/theme/foundations/colors.ts [hunk 3]
```diff
@@ -14,6 +14,10 @@ const bbBearColors = {
   },
   interaction: {
     muted: {
+      main: {
+        hover: '#fef8fb',
```
Comment (pregnantboy) [inline packages/frontend/src/theme/foundations/colors.ts]: capitalize

**Review by pregnantboy — APPROVED** [review]
looks pretty good. can the chevron be bigger tho?

File: packages/frontend/src/components/FlowSubstepTitle/index.tsx (hunk 1 above)
Comment (m0nggh) [reply packages/frontend/src/components/FlowSubstepTitle/index.tsx]: ma badd

**Pull request merged**